configure.ac

```diff
@@ -1247,6 +1247,16 @@ case "$host_os" in
 	;;
 esac
 
+dnl Visibility annotations...
+saved_CFLAGS="$CFLAGS"
+CFLAGS="$CFLAGS -fvisibility=hidden -Wall -Werror"
```

Comment:
Personally I would omit the `-Wall` since it can induce unrelated warnings (and thus issues) when detecting the feature.

Reply:
I was worried that some compilers may emit warning about unsupported visibility attribute only under `-Wall`. Do you think this is a false assumption?

Reply:
Ok, fixed in latest version of PR.